hare/algorithm_test.go

```diff
@@ -174,6 +175,44 @@ func (mev *mockEligibilityValidator) ValidateEligibilityGossip(context.Context,
 	return atomic.LoadInt32(&mev.valid) != 0
 }
 
+// We assert the the theshold X is met a long as there is one honest vote and
+// the sum of honest, dishonest and known equivocator is >= X.
```

Comment:
this description is not exactly right. the condition also requires at least 1 honest vote.

Reply:
That's in there right?

Reply:
Oh do you mean i should say *at least one* ?